apps/notification-tester/targets/notification-service/NotificationService.swift

```diff
@@ -0,0 +1,75 @@
+import UserNotifications
```

Comment:
Can this be used as it is in every project? Or should I change something?

Reply:
use as-is